victor:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  victor_left_arm_joint_1_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_left_arm_joint_1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_left_arm_joint_2_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_left_arm_joint_2
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_left_arm_joint_3_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_left_arm_joint_3
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_left_arm_joint_4_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_left_arm_joint_4
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_left_arm_joint_5_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_left_arm_joint_5
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_left_arm_joint_6_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_left_arm_joint_6
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_left_arm_joint_7_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_left_arm_joint_7
    pid: {p: 100.0, i: 0.01, d: 10.0}

  victor_right_arm_joint_1_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_right_arm_joint_1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_right_arm_joint_2_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_right_arm_joint_2
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_right_arm_joint_3_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_right_arm_joint_3
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_right_arm_joint_4_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_right_arm_joint_4
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_right_arm_joint_5_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_right_arm_joint_5
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_right_arm_joint_6_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_right_arm_joint_6
    pid: {p: 100.0, i: 0.01, d: 10.0}
  victor_right_arm_joint_7_position_controller:
    type: position_controllers/JointPositionController
    joint: victor_right_arm_joint_7
    pid: {p: 100.0, i: 0.01, d: 10.0}

  # trajectory controller
  position_trajectory_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - victor_left_arm_joint_1
      - victor_left_arm_joint_2
      - victor_left_arm_joint_3
      - victor_left_arm_joint_4
      - victor_left_arm_joint_5
      - victor_left_arm_joint_6
      - victor_left_arm_joint_7
      - victor_right_arm_joint_1
      - victor_right_arm_joint_2
      - victor_right_arm_joint_3
      - victor_right_arm_joint_4
      - victor_right_arm_joint_5
      - victor_right_arm_joint_6
      - victor_right_arm_joint_7

  # gazebo
  gazebo_ros_control:
    pid_gains:
      victor_left_arm_joint_1:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_left_arm_joint_2:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_left_arm_joint_3:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_left_arm_joint_4:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_left_arm_joint_5:
        p: 1000.0
        i: 0.01
        d: 10.0
      victor_left_arm_joint_6:
        p: 1000.0
        i: 0.01
        d: 10.0
      victor_left_arm_joint_7:
        p: 1000.0
        i: 0.01
        d: 10.0

      victor_right_arm_joint_1:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_right_arm_joint_2:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_right_arm_joint_3:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_right_arm_joint_4:
        p: 10000.0
        i: 0.01
        d: 10.0
      victor_right_arm_joint_5:
        p: 1000.0
        i: 0.01
        d: 10.0
      victor_right_arm_joint_6:
        p: 1000.0
        i: 0.01
        d: 10.0
      victor_right_arm_joint_7:
        p: 1000.0
        i: 0.01
        d: 10.0
